HISTORY.md

```diff
@@ -1,3 +1,7 @@
+### mlpack 3.0.5
+###### ????-??-??
+  * Change DBSCAN to use PointSelectionPolicy and add RandomSelectionPolicy (#1625).
```

Comment:
If I'm not wrong, haven't you implemented `OrderedPointSelection` policy instead of `RandomSelectionPolicy` which was already implemented?

Reply:
Hi there, thank you for reviewing it.
Ah!, you're right. The `OrderedPointSelectionPolicy` is newly implemented, and the `RandomSelectionPolicy` is edited for use because it is not used before. The existing `src/mlpack/methods/dbscan/dbscan_impl.hpp` have selected points sequentially. This is why I selected the `RandonSelectionPolicy`, not `OrderedSelectionPolicy`.

But, after hearing what you said, It is better to change `RandomSelectionPolicy` to `OrderedSelectionPolicy` because It is actually new. Thanks!!